lib/assessment.rb

```diff
@@ -13,28 +15,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(:+)
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.any? {|number| number.odd? }
```

Comment:
Correct. You could also use the shorthand notation that you used in Q2:
`numbers.any?(:odd?)`